packages/flutter_tools/lib/src/commands/screenshot.dart

```diff
@@ -120,6 +120,15 @@ class ScreenshotCommand extends FlutterCommand {
                    : FlutterCommandResult.fail();
   }
 
+  @override
+  Future<Device?> findTargetDevice({
```

Comment:
rather than this, we could just pass `includeDevicesUnsupportedByProject: true` at line 81, right?

Reply:
Thanks @christopherfujino. Agreed. 

I just tested it and I'll update and push the change shortly.